gentle_gnomes/src/azavea.py

```diff
@@ -1,39 +1,46 @@
-from typing import Dict, Iterator, List, NamedTuple, Optional, Union
-
-import requests
+import aiohttp
+import typing as t
+import asyncio as aio
 
 BASE_URL = 'https://app.climate.azavea.com/api'
 
 
-class City(NamedTuple):
+class City(t.NamedTuple):
     name: str
     admin: str
     id: int
 
-    def __str__(self):
+    async def __str__(self):
```

Comment:
Doesn't need to be async since it doesn't await anything. I don't think it _can_ be async anyway.